### PR TITLE
working on scrabble

### PR DESCRIPTION
**Pull request by imsolost**
ignore the converter test, it was used just to figure out how to set up a testing environment.

### CODE REVIEW

File: test/11_LairotcafTest.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var expect    = require("chai").expect;
+var lairotcaf = require("../scripts/11_Lairotcaf");
```
Comment (DianaVashti) [inline test/11_LairotcafTest.js:2]: took me a second but I appreciate the var name.

File: test/11_LairotcafTest.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+describe("reverse factorial test", function() {
+  it("should return the string 6!", function() {
+    var moddedString  = lairotcaf.reverseFactorial( 720 );
+
+    expect(moddedString).to.equal( '6!' );
+  });
+
+  it("should return the string none", function() {
```
Comment (DianaVashti) [inline test/11_LairotcafTest.js:11]: include why in test like `"Returns the string none if input has no factorial base"` or something more concise

File: test/3_stringIndicesTest.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    expect(twentyFirstWord).to.equal("");
+  });
+  it("should return '' if the number given is negative", function() {
+    var negativeOne = stringIndices.wordAtIndex('I want to go to sleep', -1);
+
+    expect(negativeOne).to.equal("");
+  });
+
+});
```
Comment (DianaVashti) [inline test/3_stringIndicesTest.js:21]: Using helper functions and var's in this way makes me distrustful of the test/code. Your test should be able to run `wordAtIndex(str, num)` on it's own as a pure function ( = same input -> same output, every time)

File: test/5_disemvowelingTest.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+var disemvoweling = require("../scripts/5_disemvoweling");
+
+describe("disemvowel test", function() {
+  it("should return the string without vowels", function() {
+    var moddedString  = disemvoweling.disemvowel('Catch me outside, how about that?');
+
+    expect(moddedString).to.equal("ctchmtsdhwbttht");
+  });
+
+});
```
Comment (DianaVashti) [inline test/5_disemvowelingTest.js:11]: see above comment about helper var's in tests

File: test/9_parens-B-goneTest.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    expect(moddedString).to.equal( '((1234)((5)67))' );
+  });
+
+  // it("should return the string with reduced parens", function() {
+  //   var moddedString  = parens_B_gone.removeParens( '12((3))' );
+  //
+  //   expect(moddedString).to.equal( '12(3)' );
+  // });
+
+});
```
Comment (DianaVashti) [inline test/9_parens-B-goneTest.js:23]: See aboveAbove || above re: helper var's in tests

File: test/converter.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      var red   = converter.hexToRgb("ff0000");
+      var green = converter.hexToRgb("00ff00");
+      var blue  = converter.hexToRgb("0000ff");
+
+      expect(red).to.deep.equal([255, 0, 0]);
+      expect(green).to.deep.equal([0, 255, 0]);
+      expect(blue).to.deep.equal([0, 0, 255]);
+    });
+  });
+});
```
Comment (DianaVashti) [inline test/converter.js]: and again... see above.

**Review by DianaVashti — CHANGES_REQUESTED** [review]
commit msg says 'working on scrabble' but the branch name is guess the number, and many files not related to scrabble are edited. I would make separate branches for each goal, especially since they do not build off eachother's code. Also, all the files edited on this PR are all tests so a better branch name would be `tests` and commit msg `Writes tests for x,y,z`

The logic behind what you are testing for is solid but the user of helper var's is a no-no. You should aim to write pure functions that handle what your current var's are doing in the scope of the function.
